### ISSUE TITLE
Languages have been removed from Android builds

### ISSUE THREAD

**Issue by IlDucci** [libretro/RetroArch#13672]
## Description

According to various sources on the Discord server, language support for at least Russian has been removed from the Android builds. Other languages could have also dissappeared.

### Expected behavior

Go to Settings -> User -> Language -> Be able to choose a different language than English.

### Actual behavior

Go to Settings -> User. The Language option is gone.

### Bisect Results

According to user master1274, this issue started happening on February 14th. I did a quick search and this PR could be the culprit of removing the LANGEXTRA feature: https://github.com/libretro/RetroArch/commit/5bacf1e9be82173ed6d547c83bc7d572f085da35 .

**Comment by GABO1423** [comment]
Indeed, that commit removed the line -DHAVE_LANGEXTRA from the phoenix-common/jni/Android.mk file. Simply restoring the line should be enough to fix this then?

**Comment by IlDucci** [comment]
Maybe. Probably. But I don't know much about programming, so I'm unsure if that was removed for any other reason that is not a mistake. Now I've realized I forgot to quote the PR's author here, so I'll do it now: @farmerbb .

**Comment by farmerbb** [comment]
The removal of `HAVE_LANGEXTRA` was originally done in #13538 which was the precursor PR to #13615.  IIRC it was needed because at the time the Android app wouldn't build without removing the flag. I just checked against master and the app does seem to build fine with the flag added back, so I submitted a PR to do that: #13676
